Packs/TaniumThreatResponse/Integrations/TaniumThreatResponse/TaniumThreatResponse.py

```diff
@@ -1242,6 +1259,56 @@ def get_process_timeline(client, data_args):
     return human_readable, outputs, raw_response
 
 
+def upload_intel_doc(client, data_args):
+    # OpenIOC format is XML
+    headers = {
+        'Content-Type': 'application/xml'
+    }
+    raw_response = client.do_request('POST', '/plugin/products/detect3/api/v1/intels/', data=data_args.get('intel_doc'),
+                                     headers=headers)
+    intel_doc = get_intel_doc_item(raw_response)
+
+    context = createContext(intel_doc, removeNull=True)
+    outputs = {'Tanium.IntelDoc(val.ID && val.ID === obj.ID)': context}
+
+    intel_doc['LabelIds'] = str(intel_doc['LabelIds']).strip('[]')
+    # trying this out instead of hardcoding, but could be out of order
+    headers = list(intel_doc.keys())
+    human_readable = tableToMarkdown('Intel Doc uploaded', intel_doc, headers=headers,
+                                     headerTransform=pascalToSpace, removeNull=True)
+
+    return human_readable, outputs, raw_response
+
+
+def start_quick_scan(client, data_args):
+    # get computer group ID from computer group name
+    computer_group_name = data_args.get('computer_group_name')
+    raw_response = client.do_request('GET', f"/api/v2/groups/by-name/{computer_group_name}")
+    raw_response_data = raw_response.get('data')
+    if not raw_response_data:
+        msg = f'No group exists with name {computer_group_name} or'
+        msg += 'your account does not have sufficient permissions to access the groups'
+        raise DemistoException(msg)
+    demisto.debug(f'data_args: {data_args}')
```

Comment:
Are you sure all arguments need to be written to log?

Reply:
No they don't need to be, we can remove this line.